### ISSUE TITLE
Add support for unsynced lyrics tag?

### ISSUE THREAD

**Issue by nensor** [devsnd/tinytag#94]
I have a preliminary change in my fork of the library that supports extraction of the USLT lyrics tag if present in mp3 files, using for my own benefit.  Should I submit a pull request for that if there's any interest in using it?

There are a number of utilities out there (e.g., Lyrics Finder, mp3tag, etc.) that find and add lyrics to mp3 files for you, which I used to enhance my own music library.  I've been tinkering with tinytag for a python search project on my own and thought I would add this to it.

**Comment by devsnd** [comment]
Yes, please, that sounds great. My only requirement would be that the lyrics are returned in a json-serializable data structure that is compatible with possibly multiple lyrics formats from different containers. (I don't know if it's just a long string, or if it has time codes, like subtitles for movies)

Generally i just want to keep tinytag as simple as possible, any post-processing of the data should be up to the users of the lib.

In short: Show me what you got! :)

**Comment by devsnd** [comment]
Hey @nensor , I did some quick investigation, and ID3 supports having multiple lyrics, each having a different language. So an mp3 cannot only contain the original lyrics, but also translated lyrics. This should be part of the returned data structure then.

See here: https://id3.org/id3v2.3.0#Unsychronised_lyrics.2Ftext_transcription

It might make sense to see what other formats support. As mentioned before, ID3 only supports unsynchronized lyrics, but other formats might support synchronized lyrics as well.

**Comment by nensor** [comment]
Yes, only looked into unsynced at the moment, and the tool(s) I was using seem to indicate an "unspecified" language code ("XXX") yet I have a number of different languages in my music files.  That said, I noted that JSON extraction using the __main__ in my testing might not properly encode for example Japanese; however, when I use csv format, I get the characters properly.

I will submit a pull request from my branch to show the very minor differences.  I'm sure the transcription is quite basic, again quite new to both mp3 tags and python in general, so forgive me.

**Comment by tcarroll2** [comment]
> It might make sense to see what other formats support. As mentioned before, ID3 only supports unsynchronized lyrics, but other formats might support synchronized lyrics as well.

Not exactly sure what you mean by the above statement.  In the link you posted, 4.9 discusses the standard for unsynchronized lyrics (tag USLT), and 4.10 discusses the standard for synchronized lyrics (tag SYLT).  I came here because someone mentioned this is the latest swiss army knife for ID3 tagging.  Unfortunately if the USLT option isn't available this library won't meet my needs.  However, thank you for the work you've done so far!

**Comment by devsnd** [comment]
Hey @tcarroll2 and @nensor 

This USLT data will be part of the `extra` field as described in #108, which can be accessed like this
```
t = TinyTag.get('audio.mp3')
print(t.extra['lyrics'])
```

You can use the feature already by using the development version in the `master` branch or you can wait for the next release.
